wiremock-main/mappings/exercise.json

```diff
@@ -53,8 +53,8 @@
         "urlPattern": "/polizas/(.*)/siniestros"
       },
       "response": {
-        "status": 200,
-        "body": "[{\"siniestroId\": \"A{{request.pathSegments.[1]}}\", \"estado\": \"En proceso\"}, {\"siniestroId\": \"B{{request.pathSegments.[1]}}\", \"estado\": \"En proceso\"}]",
+        "status": 500,
```

Comment:
Not a problem. But this way you "lose" the original response. Maybe you could've duplicated the file to expose a new scenario... and wiremock also has some other options in order to answer with different responses depending on something... But even simpler... you could've tested the issue just killing wiremock server because the service would've failed trying to contact if it's not there anymore to answer :)

Reply:
okey, I'm just killing wiremock now, makes more sense and it's easier. I had issues with the exceptions as I changed to feign client and I had to add an extra check for one exception to know if service is unavailable.